hypershift-operator/controlplaneoperator-overrides/overrides.go

```diff
@@ -0,0 +1,84 @@
+package controlplaneoperatoroverrides
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+
+	"github.com/openshift/hypershift/support/config"
+	"gopkg.in/yaml.v2"
+)
+
+type CPOOverrides struct {
+	Overrides []CPOOverride `yaml:"overrides,omitempty"`
+}
+
+type CPOOverride struct {
+	Version  string                  `yaml:"version"`
+	CPOImage string                  `yaml:"cpoImage"`
+	Testing  CPOOverrideTestReleases `yaml:"testing"`
```

Comment:
What is this field for?

Reply:
This is for our e2e testing. It specifies which release images should be used for testing this particular override.